manifests/nx-gzip/04-daemonset-non-privileged.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: ocp4-nx-gzip-power-workload
+  namespace: nx-gzip
+  labels:
+    app: power-nx-gzip-workload
+spec:
+  selector:
+    matchLabels:
+      app: power-nx-gzip-workload
+  template:
+    metadata:
+      annotations:
+        openshift.io/required-scc: privileged
+        openshift.io/scc: privileged
+      labels:
+        app: power-nx-gzip-workload
+    spec:
+      containers:
+      - name: nx-gzip-workload
+        image: quay.io/powercloud/ocp4-power-workload-tools:main
+        imagePullPolicy: IfNotPresent
+        command: [ "/bin/sleep" ]
+        args: [ "infinity" ]
+        resources:
+          limits:
+            memory: 4000Mi
+          requests:
+            cpu: 4
+            memory: 4000Mi
+        volumeMounts:
+        - name: host-sys
+          mountPath: /sys
```

Comment:
```suggestion
```
Only if the local test passes https://github.com/IBM/ocp4-power-workload-tools/pull/3/files#r1971923042